src/constants/networks.ts

```diff
@@ -107,9 +107,12 @@ export const FALLBACK_URLS = {
   ],
   [ChainId.BASE]: [
     // "Safe" URLs
-    'https://mainnet.base.org',
+    'https://mainnet.base.org/',
+    'https://developer-access-mainnet.base.org/',
     'https://base.gateway.tenderly.co',
+    'wss://base.gateway.tenderly.co',
```

Comment:
not sure if we can connect with websocket urls, can you confirm?

Reply:
i'll just remove since it's a duplicate